treeherder/webapp/api/logslice.py

```diff
@@ -38,8 +39,8 @@ def list(self, request, project, jm):
         Receives a line range and job_id and returns those lines
         """
         job_id = request.QUERY_PARAMS.get("job_id")
-
-        log = jm.get_log_references(job_id)
+        log_name = request.QUERY_PARAMS.get("name", "builds-4h")
```

Comment:
Yup that's fine - and what I meant above (though not sure if that was clear) :-)